.github/workflows/bump-channel-version/action.yml

```diff
@@ -8,6 +8,15 @@ inputs:
   CLEAN_PACKAGE_VERSION:
     required: true
     description: 'The current package version, e.g. "3.11.7".'
+  OVERRIDE_PACKAGE_VERSION:
+    required: false
+    default: "false"
+    description: 'Override the package version in the workflow environment variables.'
+  POSTFIX:
```

Comment:
I think that the `CLEAN_PACKAGE_VERSION` argument has `-rc` already appended to it

Reply:
I am not sure about that,
CLEAN_PACKAGE_VERSION=$(node -p "require('./package.json').version") taken from .github/workflows/get-new-package-version/action.yml

The -rc postfix is added in .github/workflows/get-new-package-version/action.yml
PACKAGE_VERSION=${CLEAN_PACKAGE_VERSION}-${{ inputs.CHANNEL }}-rc

What I did was to replace the fixed -rc postfix with a variable, that way we can add any postfix that we want.